docs/en/reference/types.rst

```diff
@@ -471,224 +471,223 @@ The following table shows an overview of Doctrine's type abstraction.
+|                   |               | **PostgreSQL**           | *all*   | ``TEXT``                                                 |
+|                   |               +--------------------------+         |                                                          |
+|                   |               | **Oracle**               | *all*   | ``CLOB``                                                 |
+|                   |               +--------------------------+         |                                                          |
+|                   |               | **SQLite**               |         |                                                          |
+|                   |               +--------------------------+---------+----------------------------------------------------------+
+|                   |               | **SQL Server**           | *all*   | ``VARCHAR(MAX)``                                         |
++-------------------+---------------+--------------------------+---------+----------------------------------------------------------+
+| **guid**          | ``string``    | **MySQL**                | *all*   | ``CHAR(36)`` [1]                                         |
+|                   |               +--------------------------+         |                                                          |
+|                   |               | **Oracle**               |         |                                                          |
+|                   |               +--------------------------+         |                                                          |
+|                   |               | **SQLite**               |         |                                                          |
+|                   |               +--------------------------+---------+----------------------------------------------------------+
+|                   |               | **SQL Server**           | *all*   | ``UNIQUEIDENTIFIER``                                     |
+|                   |               +--------------------------+         |                                                          |
+|                   |               | **PostgreSQL**           | *all*   | ``UUID``                                                 |
++-------------------+---------------+--------------------------+---------+----------------------------------------------------------+
+| **binary**        | ``resource``  | **MySQL**                | *all*   | ``VARBINARY(n)`` [3]                                     |
+| [2]  [6]          |               +--------------------------+         +----------------------------------------------------------+
+|                   |               | **SQL Server**           |         | ``BINARY(n)`` [4]                                        |
+|                   |               +--------------------------+         +----------------------------------------------------------+
```

Comment:
According to the discussion below, I think the right thing to do here is the following:

```suggestion
| [2]  [6]          |               +--------------------------+         |                                                          |
|                   |               | **SQL Server**           |         | ``BINARY(n)`` [4]                                        |
|                   |               +--------------------------+---------+----------------------------------------------------------+
```